golem-worker-executor-base/src/invocation.rs

```diff
@@ -485,9 +485,18 @@ async fn invoke_http_handler<Ctx: WorkerCtx>(
         }
     };
 
-    let res_or_error = match receiver.await {
+    let out = receiver.await;
+
+    tracing::warn!("received response {:?}", out);
```

Comment:
Let's remove these debug warn!s (there is another one below too)

Reply:
oh yes, sorry